specification/oep/resource-manager/Microsoft.OpenEnergyPlatform/preview/2021-06-01-preview/oep.json

```diff
@@ -0,0 +1,517 @@
+              "$ref": "#/definitions/OepResourceList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed."
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "OepResource_ListByResourceGroup": {
+            "$ref": "./examples/OepResource_ListByResourceGroup.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.OpenEnergyPlatform/energyServices": {
+      "get": {
+        "tags": [
+          "UserRPOepResource"
+        ],
+        "description": "Lists a collection of oep resources under the given Azure Subscription ID.",
+        "operationId": "OepResources_ListBySubscriptionId",
+        "x-ms-examples": {
+          "OepResource_ListBySubscriptionId": {
+            "$ref": "./examples/OepResource_ListBySubscriptionId.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returned when the operation is successful.",
+            "schema": {
+              "$ref": "#/definitions/OepResourceList"
+            }
+          },
+          "default": {
+            "x-nullable": false,
+            "description": "Error response describing why the operation failed."
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OpenEnergyPlatform/energyServices/{resourceName}": {
+      "get": {
+        "tags": [
+          "UserRPOepResource"
+        ],
+        "description": "Returns oep resource for a given name.",
+        "operationId": "OepResources_Get",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "in": "path",
+            "name": "resourceName",
+            "description": "The resource name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/OepResource"
```

Comment:
Considering the resource type of this resource is `Microsoft.OpenEnergyPlatform/energyServices`, I would suggest we rename this model to `EnergyService` to make things clear, because `Oep` actually is the name of this RP, and the name `resource of Oep` actually not telling anything to the customer

Reply:
FMU: Is that a breaking change?

Reply:
Since your swagger is not onboarded yet, it is not even a change. But if you do this after this swagger is onboarded, it is a breaking change

Reply:
Ah, I noticed that you have onboarded this on `RPaaSMaster`. I would say this is not a breaking change to the swagger, but it will be a breaking change and refinement for the SDKs.
Swaggers do not need the model names, but the SDKs need. Therefore I suggest we still do this change here.